creation/lib/cgWParamDict.py

```diff
@@ -411,10 +422,16 @@ def populate(self, other=None):
                 else:
                     # can be overridden at the entry level, so ignore and [entry supersedes global setting]
                     print(
-                        "...cvmfsexec distributions unavailable but on-demand CVMFS requested via GLIDEIN_USE_CVMFSEXEC; Continuing..."
+                        "...cvmfsexec distributions unavailable but on-demand CVMFS requested via GLIDEIN_USE_CVMFS; Continuing..."
                     )
 
         # add additional system scripts
+        for script_name in precvmfs_file_list_scripts:
+            self.dicts["precvmfs_file_list"].add_from_file(
```

Comment:
Here `precvmfs_file_list` refers to the dictionary key and not the constant (corresponding to the filenames). So this will remain the way it is and cannot be substituted with the constant.